libs/test-utils/src/mocks/accountant.rs

```diff
@@ -166,6 +166,27 @@ macro_rules! impl_mock_accountant {
 				}
 			}
 
+			#[cfg(feature = "runtime-benchmarks")]
+			impl<Tokens> cfg_traits::benchmarking::PoolBenchmarkHelper for $name<Tokens> {
+				type AccountId = $account_id;
+				type Balance = $balance;
+				type PoolId = ();
+
+				fn bench_create_pool(_: Self::PoolId, _: &Self::AccountId) {}
+
+				fn bench_investor_setup(_: Self::PoolId, _: Self::AccountId, _: Self::Balance) {}
+			}
```

Comment:
Why do we have this on the accountant and not on the `pool`?

Reply:
The actual benchmark implementation is already in the pool. But we need here too, to pass the benchmarking tests using the accountant mock (which actually mocks the pool from the accountant's perspective).

Reply:
Ideally, we could change this by the mock coming from `mock-builder`, but I had no time to refactor the tests with this